htdocs/cron/info.php

```diff
@@ -78,7 +78,10 @@
 
 print '<table width="100%"><tr><td>';
 dol_print_object_info($object);
-print '</td></tr></table>';
+print '</td></tr>';
+print '<tr><td>'.$langs->trans('CronLastResult').' : '.$object->lastresult.'</td></tr>';
```

Comment:
lastresult and lastoutput are data coming from outside. So it must be sanitized when outpout.
Can you try  dolPrintHTML($object->lastoutput);